_data/committers.yml

```diff
@@ -218,7 +218,7 @@
 - name: Wes McKinney
   role: PMC
   alias: wesm
-  affiliation: Voltron Data
+  affiliation: "-"
```

Comment:
You can add "Posit PBC" here, though I am still affiliated with Voltron Data as an advisor.